app/views/events/index.html.erb

```diff
@@ -41,8 +22,10 @@
     %>
   <% end %>
 <% else %>
-  <%= render(Events::NoResultsComponent.new) do %>
-    Sorry your search has not found any events, try a different type, location or month.
+  <% if @performed_search && @events_search_type != "222750001" %>
+    <%= render(Events::NoResultsComponent.new) do %>
+      Sorry your search has not found any events, try a different type, location or month.
```

Comment:
@leoapost is it a matter of just updating this to include the sign up link?

Reply:
@MylesJarvis we need that as is for the other event types, but you were not that far off. We just need to check if the event type is a TTT one and render the NoResultsComponent with the proper content.